software/webapp/src/main/java/brooklyn/entity/webapp/jboss/Wildfly8ServerImpl.java

```diff
@@ -0,0 +1,196 @@
+//                .poll(new HttpPollConfig<Integer>(ERROR_COUNT)
+//                        .vars(includeRuntimeUriVars)
+//                        .onSuccess(HttpValueFunctions.jsonContents("errorCount", Integer.class)))
+//                .poll(new HttpPollConfig<Integer>(TOTAL_PROCESSING_TIME)
+//                        .vars(includeRuntimeUriVars)
+//                        .onSuccess(HttpValueFunctions.jsonContents("processingTime", Integer.class)))
+//                .poll(new HttpPollConfig<Integer>(MAX_PROCESSING_TIME)
+//                        .vars(includeRuntimeUriVars)
+//                        .onSuccess(HttpValueFunctions.jsonContents("maxTime", Integer.class)))
+//                .poll(new HttpPollConfig<Long>(BYTES_RECEIVED)
+//                        .vars(includeRuntimeUriVars)
+//                        // jboss seems to report 0 even if it has received lots of requests; dunno why.
+//                        .onSuccess(HttpValueFunctions.jsonContents("bytesReceived", Long.class)))
+//                .poll(new HttpPollConfig<Long>(BYTES_SENT)
+//                        .vars(includeRuntimeUriVars)
+//                        .onSuccess(HttpValueFunctions.jsonContents("bytesSent", Long.class)))
+                        
+                .build();
+        
+        connectServiceUp();
+    }
+    
+    protected void connectServiceUp() {
+        connectServiceUpIsRunning();
+        
+        addEnricher(Enrichers.builder().updatingMap(Attributes.SERVICE_NOT_UP_INDICATORS)
+            .from(MANAGEMENT_URL_UP)
+            .computing(Functionals.ifNotEquals(true).value("Management URL not reachable") )
+            .build());
+    }
+    
+    protected void disconnectServiceUp() {
+        disconnectServiceUpIsRunning();
+    }
+    
+    @Override
+    protected void disconnectSensors() {
+        super.disconnectSensors();
+        
+        if (httpFeed != null) httpFeed.stop();
+        disconnectServiceUp();
+    }
+    
+    public int getManagementHttpsPort() {
+        return getAttribute(MANAGEMENT_HTTPS_PORT);
+    }
+    
+    public int getManagementHttpPort() {
+        return getAttribute(MANAGEMENT_HTTP_PORT);
+    }
+    
+    public int getManagementNativePort() {
+        return getAttribute(MANAGEMENT_NATIVE_PORT);
+    }
+    
+    public int getPortOffset() {
+        return getConfig(PORT_INCREMENT);
+    }
+    
+    public boolean isWelcomeRootEnabled() {
+        return false;
+    }
+
+    public String getBindAddress() {
+        return getConfig(BIND_ADDRESS);
+    }
+    
+    public String getManagementBindAddress() {
+        return getConfig(BIND_ADDRESS);
+    }
+    
+    public String getUnsecureBindAddress() {
+        return getConfig(BIND_ADDRESS);
+    }
+    
+    // If empty-string, disables Management security (!) by excluding the security-realm attribute
+    public String getHttpManagementInterfaceSecurityRealm() {
```

Comment:
Appears not to be used. I see you took it from the JBoss7 impl. Not sure what uses that one either!